### PR TITLE
Mapper bug fix

### PR DESCRIPTION
**Pull request by vtnate**
### Pull Request Description

- Bugfix in baseline mapper that wasn't reading the feature file correctly
- Update `base_workflow.osw` to not show impossible date
- Revert commits that used a custom `base_workflow.osw` file

### Checklist (Delete lines that don't apply)

- [x] All ci tests pass (green)
- [x] An [ISSUE](https://github.com/urbanopt/urbanopt-cli/issues) has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run
- [x] This branch is up-to-date with develop

### CODE REVIEW

File: example_files/mappers/Baseline.rb [hunk 1]
```diff
@@ -372,18 +372,8 @@ def create_osw(scenario, features, feature_names)
             end
             
             floor_height = 10
-            # Map system type to openstudio system types
```
Comment (kflemin) [inline example_files/mappers/Baseline.rb:375]: this doesn't look right...this bit of code 375-386 should not get removed.

Comment (vtnate) [reply example_files/mappers/Baseline.rb]: Fascinating. Must have been a branch thing, as I had copied the file from the example project. In this commit I copied `example_project.json`, `base_workflow.osw`, and `Baseline.rb` from the example project to here. Only Baseline.rb changed, restoring those lines that shouldn't have been deleted.

**Comment by vtnate** [comment]
The REopt tests still fail when run by Jenkins, though they pass when run locally. @kflemin can you confirm the tests pass locally for you?

**Comment by vtnate** [comment]
@kflemin: @tijcolem and I sorted out some shenanigans with Jenkins, and finally the REopt tests pass! The timestep fix won't be implemented until this branch gets pushed to Master, because that's where the CLI downloads the files from.

**Comment by vtnate** [comment]
I'm closing #92 now so it can be included in the changelog in this PR, to prepare for release.

**Comment by vtnate** [comment]
I suspect this failure is due to the reopt timeout, intermittently caused by the timestep bug fixed in this PR. When the change to `Baseline.rb` from this PR is merged to Master I expect these tests to pass reliably.

**Pull request merged**